src/sidebar/components/test/annotation-header-test.js

```diff
@@ -1,273 +1,155 @@
+        },
+        {
+          replyCount: 1,
+          expected: '1 reply',
+        },
+        {
+          replyCount: 2,
+          expected: '2 replies',
+        },
+      ]
+    );
   });
 
-  beforeEach('Initialize and register fake AngularJS dependencies', function() {
-    fakeFeatures = {
-      flagEnabled: sinon.stub().returns(false),
-    };
+  describe('timestamp', () => {
+    it('should render a timestamp if annotation has an `updated` value', () => {
+      const wrapper = createAnnotationHeader();
+      const timestamp = wrapper.find(Timestamp);
 
-    angular.mock.module('app', {
-      features: fakeFeatures,
-      groups: fakeGroups,
-      settings: fakeSettings,
-      serviceUrl: fakeServiceUrl,
+      assert.isOk(timestamp.exists());
     });
+    it('should not render a timestamp if annotation does not have an `updated` value', () => {
+      const wrapper = createAnnotationHeader({
+        annotation: fixtures.newAnnotation(),
+      });
+      const timestamp = wrapper.find(Timestamp);
 
-    angular.mock.inject(function(_$componentController_) {
-      $componentController = _$componentController_;
+      assert.isNotOk(timestamp.exists());
     });
   });
 
-  describe('sidebar.components.AnnotationHeaderController', function() {
-    describe('#htmlLink()', function() {
-      it('returns the HTML link when available', function() {
-        const ann = fixtures.defaultAnnotation();
-        ann.links = { html: 'https://annotation.service/123' };
-        const ctrl = $componentController(
-          'annotationHeader',
-          {},
-          {
-            annotation: ann,
-          }
-        );
-        assert.equal(ctrl.htmlLink(), ann.links.html);
+  describe('annotation is-highlight icon', () => {
+    it('should display is-highlight icon if annotation is a highlight', () => {
+      const wrapper = createAnnotationHeader({
+        isEditing: false,
+        isHighlight: true,
       });
+      const highlightIcon = wrapper.find('.annotation-header__highlight');
 
-      it('returns an empty string when no HTML link is available', function() {
-        const ann = fixtures.defaultAnnotation();
-        ann.links = {};
-        const ctrl = $componentController(
-          'annotationHeader',
-          {},
-          {
-            annotation: ann,
-          }
-        );
-        assert.equal(ctrl.htmlLink(), '');
-      });
+      assert.isOk(highlightIcon.exists());
     });
-
-    describe('#documentMeta()', function() {
-      it('returns the domain, title link and text for the annotation', function() {
-        const ann = fixtures.defaultAnnotation();
-        const ctrl = $componentController(
-          'annotationHeader',
-          {},
-          {
-            annotation: ann,
-          }
-        );
-        assert.deepEqual(ctrl.documentMeta(), fakeDocumentMeta);
+    it('should not display the is-highlight icon if annotation is not a highlight', () => {
```

Comment:
Nit: Would you mind adding new lines between tests for consistency with the rest of the file.